pkgs/games/npush/default.nix

```diff
@@ -24,6 +24,7 @@ stdenv.mkDerivation rec {
   makeFlags = [
     "CC=${stdenv.cc.targetPrefix}c++"
   ];
+  NIX_CFLAGS_COMPILE = "-Wno-error=format-security";
```

Comment:
This should be done via hardenedDisable.

Reply:
Feel free to provide a fixup.